server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 8080
+const path = require('path')
+const public = path.resolve('./public')
+
+
+app.use(express.static(public))
+
+app.listen(port, () => console.log(`Example app listening on port: ${port}`))
+
+app.get('/', (req, res) => {
+    res.sendFile(public + '/index.html')
+})
+
```

Comment:
Depending on how you generate the `hrefs` in your Restaurant List on the home page, you MAY need to figure out how to pass that restaurant's information to the server and next page.

In other words, if you have a link with the `href="/restaurant/city-market"`, you would need to set up your paths here to allow the `city-market` part of it to pass as a parameter, and NOT a direct path.

See:[Express Route Parameters](http://expressjs.com/en/guide/routing.html#route-paths)

There's no one way to do this, but something to keep in mind!